src/components/Dialog/Dialog.tsx

```diff
@@ -9,13 +10,14 @@ import ModalCloseButton from '#components/ModalCloseButton/ModalCloseButton';
 type Props = {
   open: boolean;
   onClose: () => void;
+  size?: 'small' | 'large';
   children: React.ReactNode;
 };
 
-const Dialog: React.FC<Props> = ({ open, onClose, children }: Props) => {
+const Dialog: React.FC<Props> = ({ open, onClose, size = 'small', children }: Props) => {
   return (
     <Modal open={open} onClose={onClose} AnimationComponent={Slide}>
-      <div className={styles.dialog}>
+      <div className={classNames(styles.dialog, size === 'large' && styles.largeDialog)}>
```

Comment:
To make this expendable (and a bit cleaner), you could use the size as a key as well. 

```suggestion
      <div className={classNames(styles.dialog, styles[size])}>
```

In the CSS module you would have:

```scss
.small {
  max-width: 450px; // extracted from .dialog selector
}

.large {
  max-width: 768px;
}
```